tensorflow/tensorflow.bzl

```diff
@@ -2653,6 +2653,29 @@ def gpu_py_test(
 def cuda_py_test(*args, **kwargs):
     gpu_py_test(*args, **kwargs)
 
+def trt_py_test(test_names, tags):
+    # cuda_py_test enable XLA tests by default. We can't combine XLA with TensorRT currently and should set
```

Comment:
s/We/TensorRT/
Please also reflow the comment to fill all the lines (<=80 chars) except for the last line.

Reply:
Done